### PR TITLE
Dev/bohdan h/mks 31 create class entity

### PR DESCRIPTION
**Pull request by BohdanHamurda**
Merge after PR MKS-19

### CODE REVIEW

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Models/Students.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
```
Comment (bvzayats) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Models/Students.cs]: It should be implemented in another PR.

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/obj/Debug/netcoreapp2.2/WebAPISwagger.AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+//------------------------------------------------------------------------------
```
Comment (bvzayats) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/obj/Debug/netcoreapp2.2/WebAPISwagger.AssemblyInfo.cs]: Remove files which gitignore sholud catch.

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/obj/Debug/netcoreapp2.2/WebAPISwagger.AssemblyInfoInputs.cache [hunk 3]
```diff
@@ -0,0 +1 @@
+28a0f8921133550a824a7e5bcba082df2eb1a1fd
```
Comment (bvzayats) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/obj/Debug/netcoreapp2.2/WebAPISwagger.AssemblyInfoInputs.cache]: Remove files which gitignore sholud catch.

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/obj/Debug/netcoreapp2.2/WebAPISwagger.csproj.CoreCompileInputs.cache [hunk 4]
```diff
@@ -0,0 +1 @@
+41e66fb688c73b66cdb9edef9ecf16838b271142
```
Comment (bvzayats) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/obj/Debug/netcoreapp2.2/WebAPISwagger.csproj.CoreCompileInputs.cache]: Remove files which gitignore sholud catch.

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/obj/WebAPISwagger.csproj.nuget.dgspec.json [hunk 5]
```diff
@@ -0,0 +1,81 @@
+{
```
Comment (bvzayats) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/obj/WebAPISwagger.csproj.nuget.dgspec.json]: Remove files which gitignore sholud catch.

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger.sln [hunk 6]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (bvzayats) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger.sln]: Update branch with the development

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Models/Students.cs (hunk 1 above)
Comment (BohdanHamurda) [reply Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Models/Students.cs]: Do you mean unused namespaces or smething else?

**Review by FoxLviv — APPROVED** [review]

**Review by TsisinskyjV — APPROVED** [review]

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Models/Students.cs [hunk 7]
```diff
@@ -0,0 +1,13 @@
+﻿
+namespace WebAPISwagger
+{
+    public partial class Students
```
Comment (Magfiozy) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Models/Students.cs]: Why partial?

Comment (BohdanHamurda) [reply Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Models/Students.cs]: This class was made from existing DB by EntityFramework. I did'n notice that class had modifier partial

**Review by bvzayats — APPROVED** [review]
